src/main/java/Gumball/Event.java

```diff
@@ -0,0 +1,25 @@
+package Gumball;
+
+import java.time.LocalDate;
+
+public class Event extends Task {
+    private LocalDate eventStart;
+    private LocalDate eventEnd;
+
+    /**
+     *
+     * @param desc The description of the Event to be added including name the start and end times.
+     * @throws TaskException
+     */
+    public Event(String desc) throws TaskException {
+        super("", desc);
+        try {
+            String[] section = desc.substring(6).split("/from | /to ");
```

Comment:
Array variable should have a plural name

```suggestion
            String[] sections = desc.substring(6).split("/from | /to ");
```